src/status_im/ui/screens/browser/events.cljs

```diff
@@ -121,6 +123,12 @@
    (when (< history-index (dec (count (:history browser))))
      (nav-update-browser cofx browser (inc history-index)))))
 
+(handlers/register-handler-fx
+ :browser.bridge.callback/scan-qr-code
```

Comment:
According to the guildeines events should be in `status-im.events` and call a single function from one of the module